R/zchunk_L202.water.resources.unlimited.R

```diff
@@ -31,50 +32,70 @@ module_water_L202.water.resources.unlimited_DISABLED <- function(command, ...) {
 
     # ===================================================
     # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
-    #
-    # NOTE: there are `merge` calls in this code. Be careful!
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: there are 'match' calls in this code. You probably want to use left_join_error_no_match
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # ===================================================
 
+    water_units_quantity <- "km^3"
+    water_units_price <- "1975$/m^3"
+    no_aglu_regions <- "Taiwan"
+    ag_only_water_types <- "biophysical water consumption"
+
+    #Create unlimited resource markets for water types
+    L102.unlimited_water_price_R_W_Y_75USDm3 %>%
+      group_by(year) %>%
+      filter (year %in% L102.unlimited_water_price_R_W_Y_75USDm3$year[1]) %>%
```

Comment:
@bpbond Actually, the order of the year in `L102.unlimited_water_price_R_W_Y_75USDm3` does not matter, because the file is in long year format (58 years), the intent is to select only the fields `GCAM_region_ID` and `water_type` correspond to one year since all other years are duplicate and not needed.